promotion/src/main/java/org/cryptimeleon/incentive/promotion/reward/RewardSideEffect.java

```diff
@@ -0,0 +1,8 @@
+package org.cryptimeleon.incentive.promotion.reward;
+
+import lombok.Value;
+
+@Value
+public class RewardSideEffect {
+    String name;
```

Comment:
Is this class envisioned to be able to handle stuff like "10% off" as well?

Reply:
Yes, this is just a place holder for these side effects on the basket